### DISCUSSION TITLE
New vehicle after a trade up: Still some questions.

### DISCUSSION THREAD

**Discussion by maxhac03** [teslamate-org/teslamate#3671] [Q&A] [answered]
Hi.

Even after searching i only have half my questions answered.

It's not a 2nd car added to my account but a replacement/trade up.

I will soon get a new Tesla and plan to keep the old car's data in Grafana and log the new car's data in the same DB/instance.
From what i can see is that after restarting the Docker container after receiving the new car the new one will get a new ID and the old one will stay there and i could choose which one i want to see in Grafana.

However, i'm guessing that in the TeslaMate main page (Were we can see live stats) the new one will replace the old one as the old one will no longer be in my account? Nothing else to do?

Thanks.

**Comment by maxhac03** [comment]
Bump to appear on top.
(Sorry).

**Reply by cwanja** [reply, marked as answer]
Never tested this, forgive me for speculating. I do believe your old vehicle will be removed from _TeslaMate_ home page once it is removed from your **Tesla account**. As long as you do not remove it from the database, _Grafana_ will continue to pull the data and display it allowing you to swap between vehicles in the drop-downs on dashboards.

**Reply by maxhac03** [reply]
Pretty much what i was expecting. i will update once i get my new car.
